lib/tasks/client_generate.rake

```diff
@@ -61,15 +61,22 @@ class ClientGenerator
 
     msg("\nGenerating API Ruby Client ...")
     generate_yaml_file(openapi_file, openapi_yaml_file)
-    system("java -jar #{generator_cli_jar} generate -i #{openapi_yaml_file} -c #{generator_config} -g ruby -o #{client_dir}")
+    system("java -jar #{generator_cli_jar} generate -i #{openapi_yaml_file} -c #{generator_config} -g #{lng} -o #{client_dir}")
   end
 end
 
 namespace :client do
-  desc "Generate the Toplogical Inventory API Ruby Client"
+  desc "Generate the Topological Inventory API Ruby Client"
   task :generate, [:client_dir] => [:environment] do |_task, args|
     default_client_dir = Pathname.new(Rails.root.join("..", "topological_inventory-api-client-ruby"))
     args.with_defaults(:client_dir => default_client_dir)
-    ClientGenerator.new.generate_ruby_client(args[:client_dir])
+    ClientGenerator.new.generate_client(args[:client_dir], "ruby")
+  end
+
+  desc "Generate the Topological Inventory API Bash Client"
```

Comment:
I'd personally just use curl, but yea I think this is a great idea. Can you see if we can use the same rake task just add an optional lang argument defaulting to ruby.  This way we can simply use the same rake task for any other language without having to add a rake task for each ?  Thanks.

Reply:
@abellotti now it defaults to ruby, but second param can be added as language

Reply:
Nice update!! Thanks @slemrmartin 

Can you just update the usage in the comment section ? that'll great.
#        bundle exec rake client:generate[/alternate_ruby_client_dir]

Thanks.

Reply:
yes, and I'll create similar pr to sources-api too.

Reply:
done